src/components/Calculator.js

```diff
@@ -1,27 +1,40 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+// Child component
+function CalculatorButton({ className, text, id }) {
+  return (
+    <button type="button" className={className} id={id}>
+      {text}
+    </button>
+  );
+}
```

Comment:
[OPTIONAL] Nice job splitting your codebase into sub-components 😉. Kindly remember that the reason this is done is for the purpose of separation of concerns and reusability😉 which makes your code maintainable and reasuable😉. Hence, I will recommend you take this component to a separate file as this is not just only professional but also strongly fulfills the purpose of separation of concerns making your codebase easily usable and maintainable👍.